src/style.css

```diff
@@ -0,0 +1,390 @@
+}
+
+.clicked {
+  border: 1px solid #20ce99;
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
+  border-bottom-left-radius: 4px;
+  border-bottom-right-radius: 4px;
+}
+
+.clicked:focus {
+  border: 1px solid #20ce99;
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
+  border-bottom-left-radius: 0px;
+  border-bottom-right-radius: 0px;
+}
+
+main {
+  padding-right: 24px;
+  padding-left: 24px;
+  border-bottom: 1px solid #f5f5f6;
+}
+
+.fields {
+  padding: 12px 0px;
+}
+
+input {
+  font-family: 'DINProRegular';
+  height: 34px;
+  border: 1px #fafafa solid;
+  color: #171718;
+  border-radius: 4px;
+  background-color: #fafafa;
+  text-align: right;
+  font-size: 13px;
+  margin: auto 0px;
+  padding: 0px 13px;
+  -o-transition:.2s;
+  -ms-transition:.2s;
+  -moz-transition:.2s;
+  -webkit-transition:.2s;
+  transition:.2s;
+}
+
+input::placeholder {
+  color: #dcdcdd;
+}
+
+input:hover {
+  box-shadow: 0 0 0pt 1pt #d1d1d3;
+  outline: none;
+  background-color: #ffffff;
+}
+
+input:focus {
+  box-shadow: 0 0 0pt 1pt #20ce99;
+  outline: none;
+  background-color: #ffffff;
+}
+
+.stopprice label,
+.limitprice label,
+.shares label {
+  display: flex;
+  justify-content: space-between;
+  height: 48px;
+}
+
+.stoppriceLabel, 
+.limitpriceLabel, 
+.sharesLabel {
+  margin: auto 0px;
+}
+
+.marketprice {
+  padding-right: 12px;
+}
+
+.marketprice label,
+.expiration label {
+  display: flex;
+  justify-content: space-between;
+  height: 48px;
+  align-items: center;
+}
+
+.marketpriceLabel,
+.marketpriceAmount,
+.expirationLabel {
+  font-weight: 600;
+  margin: auto 0px;
+}
+
+.focusprice {
+  color: #20ce99;
+}
+
+.__control {
+  position: relative;
+  border: 1px solid #fafafa;
+  border-radius: 4px;
+  background-color: #fafafa;
+  width: 128px;
+  justify-content: flex-start;
+  padding: 0px 13px;
+  height: 34px;
+  display: flex;
+  align-items: center;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  -o-transition:.2s;
+  -ms-transition:.2s;
+  -moz-transition:.2s;
+  -webkit-transition:.2s;
+  transition:.2s;
+}
+
+.notclicked {
+  border: 1px solid #fafafa;
+}
+
+.notclicked:hover {
+  border: 1px solid #d1d1d3;
+  background-color: #ffffff;
+  border-radius: 4px;
+}
+
+.__options {
+  position: absolute;
+  margin-top: -1px;
+  margin-left: -1px;
+  display: flex;
+  list-style-type: none;
+  border: 1px solid #20ce99;
+  border-bottom-left-radius: 4px;
+  border-bottom-right-radius: 4px;
+  background-color: #fafafa;
+  color: #d9d9d9;
+  width: 156px;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  align-content: center;
+  padding: 0px 0px;
+}
+
+.__item {
+  padding: 8px 13px;
+  cursor: pointer;
+  margin: auto 0px;
+  width: 130px;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
```

Comment:
[Cross browser compatibility](https://en.wikipedia.org/wiki/Cross-browser_compatibility) for user-select